app/src/main/java/de/danoeh/antennapod/fragment/ItemDescriptionFragment.java

```diff
@@ -113,6 +115,7 @@ private void load() {
     public void onPause() {
         super.onPause();
         savePreference();
+        webvDescription.scrollTo(0, 0);
```

Comment:
>  If the screen is turned off theres no scrollling because backToCover is called.

I don't think we should really do this, as it breaks the use-case of turning off the screen for a while and continuing to read at the same position. Another use-case would be that you click a link in the shownotes but when you come back from the web browser, the app will lose the scroll position.

Reply:
well i think we should keep this, without it there is some pretty weird behaviour (when switching episode with opened description and so on), i tested this, also if you leave the app and return, the CoverFragmet is shown anyway (as it should), also i belive  reading _description - turning of screen - continue reading_ is a fringe use case at best and its only a minor inconvenience to use scroll down to the description once more. The most user want notice this at all, but they would notice if the description is unexpectedly scrolled halfway.

Reply:
i do see that this is producing a uggly jump is a link is pressed...let me see

Reply:
i see that there's a `resore scroll position` mechanism in restorePreferences, i thought that was a bug...i dont know if this is a good feature, i think is weird if you scroll down from the cover fragment and end up in the middle of the description....